README.rst

```diff
@@ -8,7 +8,7 @@ trading gear for hackers.
     :target: https://actions-badge.atrox.dev/piker/pikers/goto
 
 ``piker`` is a broker agnostic, next-gen FOSS toolset for real-time
-trading targeted at hardcore Linux users.
+trading targeted at retail traders (pikers) who are not satisfied with the market tools that are currently available.
```

Comment:
drop both *retail* and *(pikers)*

Reply:
maybe s/*with the market tools that are currently available.*/*current market tools*?